docs/sql/ddl.txt

```diff
@@ -812,3 +812,63 @@ node startup defined configuration file value or to its default value::
     +--------------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+
+Show Create Table
+=================
+
+.. hide:
+    cr> create table if not exists my_table (
+    ... first_column integer primary key,
+    ... second_column string,
+    ... third_column timestamp
+    ... ) clustered by (first_column) into 5 shards;
+    CREATE OK (... sec)
+
+The ``SHOW CREATE TABLE`` statement can be used to print the DDL statement of
+already existing user-created doc table in the cluster::
+
+    cr> show create table my_table;
+    +-----------------------------------------------------+
+    | SHOW CREATE TABLE my_table                          |
+    +-----------------------------------------------------+
+    | CREATE TABLE IF NOT EXISTS "my_table" (             |
+    |    "first_column" INTEGER,                          |
+    |    "second_column" STRING,                          |
+    |    "third_column" TIMESTAMP,                        |
+    |    PRIMARY KEY ("first_column")                     |
+    | )                                                   |
+    | CLUSTERED BY ("first_column") INTO 5 SHARDS         |
+    | WITH (                                              |
+    |    "blocks.metadata" = false,                       |
+    |    "blocks.read" = false,                           |
+    |    "blocks.read_only" = false,                      |
+    |    "blocks.write" = false,                          |
+    |    "gateway.local.sync" = 5000,                     |
+    |    number_of_replicas = '1',                        |
+    |    "recovery.initial_shards" = 'quorum',            |
+    |    refresh_interval = 1000,                         |
+    |    "routing.allocation.enable" = 'all',             |
+    |    "routing.allocation.total_shards_per_node" = -1, |
+    |    "translog.disable_flush" = false,                |
+    |    "translog.flush_threshold_ops" = 2147483647,     |
+    |    "translog.flush_threshold_period" = 1800000,     |
+    |    "translog.flush_threshold_size" = 209715200,     |
+    |    "translog.interval" = 5000,                      |
+    |    "warmer.enabled" = true                          |
+    | )                                                   |
+    +-----------------------------------------------------+
+    SHOW 1 row in set (... sec)
+
+It is not possible to invoke ``SHOW CREATE TABLE`` on blob tables
```

Comment:
not sure about this, but I think I'd expect that kind of information in the reference instead of here.